tests/test_torch_agent.py

```diff
@@ -12,6 +12,7 @@
 from parlai.core.testing_utils import capture_output, tempdir
 from parlai.core.utils import Message
 import parlai.core.testing_utils as testing_utils
+from parlai.core.logging_utils import logger
```

Comment:
Do we need the import in these places that won't use it?

Reply:
Not sure why I imported it there, you're right it's not being used and I will remove that!